pkg/resourceutil/sort.go

```diff
@@ -14,10 +14,18 @@
 
 package resourceutil
 
-import "sort"
+import (
+	"sort"
+	"strings"
+)
 
 type resourceOrder []string
 
+// Annotation to override resource application order,
+// it should be a comma separated list of kinds for which
+// this specific resource must be applied before.
```

Comment:
may you add an example? here, and in the docs

Reply:
Done